free-programming-books-ja.md

```diff
@@ -228,7 +228,7 @@
 * [C++入門](http://www.asahi-net.or.jp/~yf8k-kbys/newcpp0.html) - 小林健一郎
 * [ロベールのＣ＋＋教室](http://www7b.biglobe.ne.jp/~robe/cpphtml/) - ロベール
 * [cpprefjp - C++ Reference Site in Japanese](http://cpprefjp.github.io)
-* [C++11の文法と機能(C++11: Syntax and Feature)](https://ezoeryou.github.com/cpp-book/C++11-Syntax-and-Feature.xhtml) - 江添亮
+* [C++11の文法と機能(C++11: Syntax and Feature)](http://ezoeryou.github.io/cpp-book/C++11-Syntax-and-Feature.xhtml) - 江添亮
```

Comment:
Done in https://github.com/davisonio/awesome-free-programming-books/commit/b3918b4b9aab2df3be45c419398722bd6c98ae6f